### PR TITLE
Feature/hide alpha banner

### LINKED ISSUE #415: Remove Alpha Banner when flag specified

### PR DESCRIPTION
**Pull request by craigzour**
# Summary | Résumé

- New `displayAlphaBanner` property has been added to JSON form template. 

Notes: If the property does not exist the Alpha banner will automatically be displayed.

closes #415

Form sample:
`{
  "form": {
    "layout": [
      1,
      2,
      3
    ],
    "titleEn": "Performance Testing Form clem",
    "titleFr": "Formulaire Test de Performance clem",
    "version": 1,
    "elements": [
      {
        "id": 1,
        "type": "richText",
        "properties": {
          "titleEn": "Info",
          "titleFr": "Info",
          "charLimit": 1000,
          "validation": {
            "required": false
          },
          "descriptionEn": "Please provide us with your contact information so that we can mail your documents to you.",
          "descriptionFr": "Veuillez nous fournir vos coordonnées afin que nous puissions vous envoyer vos documents",
          "placeholderEn": "",
          "placeholderFr": ""
        }
      },
      {
        "id": 2,
        "type": "textField",
        "properties": {
          "titleEn": "Email",
          "titleFr": "",
          "charLimit": 100,
          "validation": {
            "required": false
          },
          "description": "",
          "placeholderEn": "",
          "placeholderFr": ""
        }
      },
      {
        "id": 3,
        "type": "dropdown",
        "properties": {
          "choices": [
            {
              "en": "Alberta",
              "fr": "Alberta"
            },
            {
              "en": "British Columbia",
              "fr": "Colombie-Britannique"
            }
          ],
          "titleEn": "Province",
          "titleFr": "Province",
          "validation": {
            "required": true
          },
          "description": "",
          "placeholderEn": "",
          "placeholderFr": ""
        }
      }
    ]
  },
  "submission": {
    "vault": true
  },
  "internalTitleEn": "Performance Testing Form 1",
  "internalTitleFr": "Formulaire Test de Performance 1",
  "publishingStatus": false,
  "displayAlphaBanner": false
}`

### CODE REVIEW

File: components/globals/Base.test.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  titleEn: "Test Form",
+  titleFr: "Formulaire de test",
+  layout: [1, 2],
+  elements: [],
+};
+
+describe("Generate the Base structure of a page", () => {
+  afterEach(cleanup);
+
+  test("Alpha banner should be displayed if form property is set to true", () => {
```
Comment (Moro-Code) [inline components/globals/Base.test.js:18]: Good job adding unit tests here !

File: lib/dataLayer.tsx [hunk 2]
```diff
@@ -229,6 +231,7 @@ async function _getFormByStatus(
                 formID: record.formID,
                 ...record.formConfig.form,
                 publishingStatus: record.formConfig.publishingStatus,
+                displayAlphaBanner: record.formConfig.displayAlphaBanner ?? true,
```
Comment (Moro-Code) [inline lib/dataLayer.tsx:234]: there probably isn't a need to explicitly set it IMO. You could treat undefined as true. Not a big deal though

**Review by Moro-Code — COMMENTED** [review]
Great work! Could we add in a cypress test as well ?

**Review by Moro-Code — APPROVED** [review]
LGTM after discussion. We need to change our cypress tests to mock response rather than actually getting / submitting to staging

**Comment by craigzour** [comment]
Created #422

**Pull request merged**